### ISSUE TITLE
server profile/profile template ilo configuration

### ISSUE THREAD

**Issue by rismoney** [HewlettPackard/terraform-provider-oneview#293]
Now using rest2600

Would like to be able to manage 'ilo hostname' and
Would like to be able to manage "directory configuration" on server profiles/profile templates.

From the restapi docs related to the 'management processor'

```
"settingType": "Directory",
                "args": {
                   "directoryAuthentication": "defaultSchema",
                   "directoryGenericLDAP": false,
                   "directoryServerAddress": "ldap.example.com",
                   "directoryServerPort": 636,
                   "directoryServerCertificate": "-----BEGIN CERTIFICATE-----\nMIIBozCCAQwCCQCWGqL41Y6YKTANBgkqhkiG9w0BAQUFADAWMRQwEgYDVQQDEwtD\nb21tb24gTmFtZTAeFw0xNzA3MTQxOTQzMjZaFw0xODA3MTQxOTQzMjZaMBYxFDAS\nBgNVBAMTC0NvbW1vbiBOYW1lMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCf\nCNTrU4AZF044Rtu8jiGR6Ce1u9K6GJE+60VCau2y4A2z5B5kKA2XnyP+2JpLRRA8\n8PjEyVJuL1fJomGF74L305j6ucetXZGcEy26XNyKFOtsBeoHtjkISYNTMxikjvC1\nXHctTYds0D6Q6u7igkN9ew8ngn61LInFqb6dLm+CmQIDAQABMA0GCSqGSIb3DQEB\nBQUAA4GBAFVOQ8zXFNHdXVa045onbkx8pgM2zK5VQ69YFtlAymFDWaS7a5M+96JW\n2c3001GDGZcW6fGqW+PEyu3COImRlEhEHaZKs511I7RfckMzZ3s7wPrQrC8WQLqI\ntiZtCWfUX7tto7YDdmfol7bHiaSbrLUv4H/B7iS9FGemA+nrghCK\n-----END CERTIFICATE-----",
                   "directoryUserContext": [
                      "OU=US,OU=Users,OU=Accounts,dc=Subdomain,dc=example,dc=com",
                      "ou=People,o=example.com"
                   ],
                   "iloObjectDistinguishedName" : "service",
                   "password" : "Passw0rd",
                }
```

**Comment by nabhajit-ray** [comment]
Thanks Rismoney. We will be looking into this.

**Comment by AsisBagga** [comment]
@rismoney Hi Rich, 
This issue has been fixed in v6.2.0, and, therefore, I will be proceeding to close this issue. 
If you have any other concerns please feel free to open a new issue. Thank you.

**Comment by rismoney** [comment]
this is great news!  appreciate all your hard work!!!
